tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -107,6 +117,41 @@ def test_write(self):
                                   topic=self.topic,
                                   partition_idx=0,
                                   count=6)
-
         self.start_consumer()
         self.run_validation()
+
+
+class EndToEndShadowIndexingTestWithFailures(EndToEndShadowIndexingBase):
+    override_default_topic_replication = EndToEndShadowIndexingBase.num_brokers
+
+    @cluster(num_nodes=5,
+             log_allow_list=RESTART_LOG_ALLOW_LIST,
```

Comment:
You'll probably find you need CHAOS_LOG_ALLOW_LIST here (although awesome if it works without it)

Reply:
I just ran into a could not parse header - will update the list